packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts

```diff
@@ -224,6 +224,14 @@ export class SummarizerNode implements IRootSummarizerNode {
                 this.refreshLatestSummaryFromPending(proposalHandle, maybeSummaryNode.referenceSequenceNumber);
                 return { latestSummaryUpdated: true, wasSummaryTracked: true };
             }
+
+            this.defaultLogger.sendTelemetryEvent({
+                eventName: "PendingSummaryNotFound",
+                proposalHandle,
+                summaryRefSeq,
+                size: this.pendingSummaries.size,
```

Comment:
I don't have a strong opinion on this, but most telemetry use `length` as the property name for similar stuff

Reply:
Same here - I saw there was an existing field called size and I used it. I will change to length if it is more used.